### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility]High Contrast][MAS4.3.1]Eclipse][Wipro]:Subscription ID is not visible for selected subscription  in "Your Subscriptions" window in hcblack mode.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1419]
**Repro Steps:**

1. Open and Start Eclipse.
2. Open your project with Eclipse.
3. Press "Alt+T" to open  Tools menu, Navigate to  "Azure", and then select "Azure Sign In".
4. In the Azure Sign In window, select "Interactive" radio button, and navigate to "Sign In" button and select it.
5. In the Azure Login Dialog window, enter your "Azure credentials", and then select "Sign in" button.
6. In "Your Subscriptions "dialog , tab to any subscription and select requires subscription then observe subscription id for selected subscription in HCBlack mode

**Expected Result:**
Subscription ID should be visible with black text in HC black mode 

**Actual Result:**
Subscription ID is not visible  in HC black mode 

Platform:
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version**: Version 1709(Os Build 16299.192)

![subscription id is not visible in HCBlack in your subscriptions window](https://user-images.githubusercontent.com/22956130/35717550-f1aa93c4-0805-11e8-9b92-1113d852d816.jpg)

**Comment by Eskibear** [comment]
Determined by Eclipse IDE, cannot be fixed here.

**Comment by Eskibear** [comment]
fixed in #1601

**Comment by WiproPSAccTest** [comment]
Verified the issue in V.3.6.0. Issue got fixed.
Please add the closed tag
![1419-fixed](https://user-images.githubusercontent.com/22956130/39759823-4eefd37e-52f1-11e8-99b0-b22ff147862f.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749869
